specification/hardwaresecuritymodules/resource-manager/Microsoft.HardwareSecurityModules/preview/2024-06-30-preview/cloudhsm.json

```diff
@@ -0,0 +1,1510 @@
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      },
+      "required": [
+        "privateLinkServiceConnectionState"
+      ],
+      "description": "Properties of the private endpoint connection."
+    },
+    "PrivateEndpointConnectionProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "The current provisioning state.",
+      "enum": [
+        "Succeeded",
+        "Creating",
+        "Deleting",
+        "Failed",
+        "Updating",
+        "InternalError",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "PrivateEndpointConnectionProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "PrivateEndpointConnectionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Array of private endpoint connections.",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of private endpoint connections."
+        }
+      },
+      "description": "List of private endpoint connections associated with the specified resource."
+    },
+    "BackupRequestProperties": {
+      "type": "object",
+      "description": "Backup properties",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupRestoreRequestBaseProperties"
+        }
+      ]
+    },
+    "RestoreRequestProperties": {
+      "type": "object",
+      "description": "Cloud Hsm Cluster restore information",
+      "properties": {
+        "backupId": {
+          "type": "string",
+          "description": "An autogenerated unique string ID for labeling the backup. It contains both a UUID and a date timestamp."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupRestoreRequestBaseProperties"
+        }
+      ]
+    },
+    "BackupRestoreRequestBaseProperties": {
+      "type": "object",
+      "description": "Backup and Restore Common properties",
+      "properties": {
+        "azureStorageBlobContainerUri": {
+          "type": "string",
+          "description": "The Azure blob storage container Uri which contains the backup",
+          "format": "uri"
+        },
+        "token": {
+          "type": "string",
+          "description": "The SAS token pointing to an Azure blob storage container. This property is reserved for Azure Backup Service."
+        }
+      },
+      "required": [
+        "azureStorageBlobContainerUri"
+      ]
+    },
+    "BackupResult": {
+      "type": "object",
+      "properties": {
+        "azureStorageBlobContainerUri": {
+          "type": "string",
+          "description": "The Azure blob storage container Uri which contains the backup",
+          "format": "uri"
+        },
```

Comment:
just to be clear , the secret part of the sas uri is not present here right ?

Reply:
yess, we do not return the SAS token in the result, just the blob storage account and the container: "https://myaccount.blob.core.windows.net/container/container",